static/app/views/dashboardsV2/detail.tsx

```diff
@@ -412,7 +412,7 @@ class DashboardDetail extends Component<Props, State> {
       return;
     }
 
-    if (!isEqual(activeFilters, dashboard.filters?.release)) {
+    if (!isEqual(activeFilters, dashboard.filters)) {
```

Comment:
Do we need to push both release and release id to the url? Also, pushing release id propagates the release id to the query string appended to every widget ends up looking like this: 
```
query: release:[frontend@22.8.0.dev0+4f88b7ae4d69906232d6584d56097ea0c98a8c8a,backend@22.8.0.dev0+4f88b7ae4d69906232d6584d56097ea0c98a8c8a,backend@22.8.0.dev0+53a13ef1ddf8c5af0a88e6c3ae51921be2545465,latest,backend@22.8.0.dev0+08535455ff80873d4c3c8d7e3bb2e821764f9dab] releaseId:[322,320,319,317,latest] 
```

Reply:
Keeping release ID in the URL as we discussed, but just omitting it before doing the query.